src/resolvers/setExpressRoutes.js

```diff
@@ -132,6 +132,10 @@ const createNestedRoutes = async (
     (relation) =>
       relation.model === model.name && relation.type === RELATIONSHIPS.HAS_MANY
   );
+  
+  if (model.instance.ignore) {
```

Comment:
One last thing, can you put the if block to the `createRouteByModel` function, please?

Reply:
> One last thing, can you put the if block to the `createRouteByModel` function, please?

Surely :)